VerifySignature.Console/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Data;
+
+namespace VerifySignature.Console
+{
+    static class Program
+    {
+        static void Main(string[] args)
+        {
+            var returnCode = MsiTools.VerifyMsiSignature(args[0], args[1]);
+            if (returnCode != CertificateCheckReturnCodes.Success)
+            {
+                throw new DataException(returnCode.ToString());
+            }
+        }
+    }
+}
```

Comment:
why do we need a separate `.Console` project that contains only this duplicate main entry point?

Reply:
Thank you so much for accepting my PR :smile: 
My idea is to have this signature verification functionality as a dotnet global tool (that is why I have the console project), where it is easy for people to install/upgrade/uninstall the tool. So with the upgrade, people do get a tool, a shared library or they can still copy the 2 files. I could with another PR improve the documentation of the project, but I cannot give an ETA for it.
In our setup, I use it like this:
dotnet tool install --global verify-signature
verify-signature <dll> <dns name>
Sorry for my delayed message - I just came back from vacation!